e2e/kubelet/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+
+	"github.com/sanity-io/litter"
+)
+
+func main() {
+	if err := run(); err != nil {
+		panic(err)
+	}
+}
+
+func run() error {
+	k8sVersion := os.Getenv("KUBE_BINARY_VERSION")
```

Comment:
validation? i guess the binary exec fails?

Reply:
done